crates/iota-graphql-e2e-tests/tests/call/simple.exp

```diff
@@ -25,13 +25,19 @@ task 4, line 32:
 //# view-object 0,0
 Owner: Account Address ( validator_0 )
 Version: 1
-Contents: iota_system::validator_cap::UnverifiedValidatorOperationCap {
+Contents: iota_system::staking_pool::StakedIota {
```

Comment:
nit: @valeriyr so this change is because the `0` object on the validator address is now the StakedIota instead of the ValidatorOperationCap. Shall we modify the original simple.move and add `view-object` calls to exhaust the objects that should be owned by `validator_0` in this test?